index.html

```diff
@@ -2,15 +2,66 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="./style.css" />
+    <script src="./hamburger.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <style>
+        </style>
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <div class="grid-container">
+        <header>
+            <div class="mobile-container">
+                <div class="topnav">
+                    <a href="#home" class="active"></a>
+                    <div id="hamburger">
```

Comment:
Mobile menu doesn't seem to be working now

Reply:
https://compassionate-boyd-ced72d.netlify.app/
<img width="97" alt="Screenshot 2021-12-04 at 19 51 58" src="https://user-images.githubusercontent.com/64493498/144722866-69266b0a-2949-4060-999d-312d213c0f94.png">